demos/ns-inputter--checkbox/index.html

```diff
@@ -0,0 +1,16 @@
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+</head>
+
+<body>
+  <main class="ndsn">
+    <ns-inputter mask="" separator="" validation="[&quot;isRequired&quot;]" heading="What would you like?" helper="" name="gas" value="">
```

Comment:
Attributes `mask` and `separator` are not available for checkboxes.
```suggestion
    <ns-inputter validation="[&quot;isRequired&quot;]" heading="What would you like?" helper="" name="gas" value="">
```